webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require("path");
+const pkg = require('./package.json')
+const webpack = require('webpack')
+
+module.exports = [
+    {
+        entry: path.join(__dirname, pkg.source),
+        mode: 'none',
+        output: {
+            path: path.join(__dirname, "dist"),
+            filename: "oreid-browser.js",
+            libraryTarget: 'umd',
```

Comment:
There is a warning in the webpack config

Warning
Please use output.library.type instead as we might drop support for output.libraryTarget in the future.

I beleive you should use something like this:
      library: {
        name: 'oreidJs',
        type: 'umd',
      },

Reply:
This warning is not showing for me. How are you having it?
<img width="789" alt="Captura de Tela 2022-08-24 às 11 05 44" src="https://user-images.githubusercontent.com/1014611/186439555-b11e96f9-4dd4-4af4-bf4f-c9a8fa01d09e.png">

I was getting another warning (not to re-export "version")

Reply:
It's in the documentation - https://webpack.js.org/configuration/output/#outputlibrarytarget

Reply:
I copied this from the webpack page...
Anyway, thanks for show me this.